app/uk/gov/hmrc/bindingtariffclassification/controllers/CaseController.scala

```diff
@@ -44,6 +44,7 @@ class CaseController @Inject()(appConfig: AppConfig,
       for {
         r <- caseService.nextCaseReference
         c <- caseService.insert(caseRequest.toCase(r))
+        _ <- caseService.addInitialSampleStatusIfExists(c)
```

Comment:
Not sure about calling methods just for their side effects.   Void methods now seem "odd" after doing scala for a few months; but see what the other devs think.

Reply:
is a good point Steve but in this case it looks ok.(humble opinion). As long as the unit test verifies that we are calling this method just once with the expected params...